packages/components/src/internal/components/buttons/ActionButton.spec.tsx

```diff
@@ -51,10 +55,4 @@ describe('<ActionButton />', () => {
         expect(onClick).toHaveBeenCalledTimes(0);
         expect(wrapper).toMatchSnapshot();
     });
-
-    test('With title', () => {
```

Comment:
Likewise, I assume there is now an orphaned snapshot in `ActionButton.spec.tsx.snap` for this that should be removed.

Reply:
Done!